packages/cli/tests/cli/add/subgraph/src/mapping.ts

```diff
@@ -0,0 +1,21 @@
+import {
```

Comment:
don't think we need this either, since we are just asserting the `.json`

Reply:
`codegen` after `add` fails if the mappings file does not exists